### PR TITLE
if user hasn't logged out, is taken to dashboard instead of marketing

### PR DESCRIPTION
**Pull request by diaz4674**
# Description

If user goes to marketing page and hasn't logged out, they will be taken to their dashboard. Works by checking if auth0 cookie is still persisting or not.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change status
- [ ] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

# Checklist:
Everything in this checklist is required for each PR.  Please do not approve a PR that does not have all of these items.  

- [x] I have pulled from master before submitting this PR
- [] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer (the peer review to approve a PR counts.  The reviewer must download and test the code)
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] There are no merge conflicts

### CODE REVIEW

File: web-workout-tracker/workout/src/components/Marketing.js [hunk 1]
```diff
@@ -17,9 +17,15 @@ const Marketing = (props) => {
   const { user } = useAuth0();
 
   React.useEffect(() => {
-    if(user !== undefined){
+    // if(document.cookie.indexOf( 'auth0.is.authenticated' ) != -1){
+    //   console.log('works')
+    // }
+    if(document.cookie.indexOf( 'auth0.is.authenticated' ) == -1){
```
Comment (TeamWorkoutTracker) [inline web-workout-tracker/workout/src/components/Marketing.js]: Maybe === will be better in this case?

Comment (nia-ja) [inline web-workout-tracker/workout/src/components/Marketing.js]: Maybe === will be better in this case?

File: web-workout-tracker/workout/src/views/landing/Landing.js [hunk 2]
```diff
@@ -26,8 +26,8 @@ class Landing extends React.Component {
     //Toggles navbar component prop to render new workout and settings button
     this.props.isRegistered()
 
-    //checks to see if user is logged in by checking local storage to render components
-    if(this.props.thisUser.user_id){
+    //checks to see if user is logged in by checking cookie to render components
+    if(document.cookie.indexOf( 'auth0.is.authenticated' ) != -1){
```
Comment (nia-ja) [inline web-workout-tracker/workout/src/views/landing/Landing.js]: And same in here but !==

**Comment by nia-ja** [comment]
But overall it looks clean for me

File: web-workout-tracker/workout/src/components/Marketing.js (hunk 1 above)
Comment (diaz4674) [reply web-workout-tracker/workout/src/components/Marketing.js]: For some reason it doesnt work with the ===, that was my original idea

**Pull request merged**